lib/src/widget/close_button.dart

```diff
@@ -6,23 +8,27 @@ class NeumorphicCloseButton extends StatelessWidget {
   final VoidCallback onPressed;
   final NeumorphicStyle style;
   final EdgeInsets padding;
+  final bool reversedIcon;
 
   const NeumorphicCloseButton({
     Key key,
     this.onPressed,
     this.style,
     this.padding,
+    this.reversedIcon = false,
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
+    final nThemeIcons = NeumorphicTheme.of(context).current.appBarTheme.icons;
     return NeumorphicButton(
       style: style,
       padding: padding,
       tooltip: MaterialLocalizations.of(context).closeButtonTooltip,
-      child: Icon(
-        Icons.close,
-      ),
+      child: reversedIcon
+          ? Transform.rotate(
```

Comment:
same feeling here not a fan of adding transformation for that... let's keep it simple of if people need more they can just do it in their app. Not sure a close button will ever need to be reversed.

Reply:
Yep, need delete